cookbooks/sidekiq6/templates/default/sidekiq.service.erb

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Sidekiq Worker <%= @count %>
+
+
+[Service]
+Type=notify
+WorkingDirectory=/data/<%= @app_name %>/current
+ExecStart=/bin/bash -l -c "bundle exec sidekiq -e <%= @rails_env %> -C /data/<%= @app_name %>/current/config/sidekiq_<%= @count %>.yml"
+
+User=root
+Group=root
+RestartSec=30
+Restart=on-failure
+
+StandardOutput=file:/data/<%= @app_name %>/current/log/sidekiq_<%= @count %>.log
+StandardError=file:/data/<%= @app_name %>/current/log/sidekiq_<%= @count %>.stderr.log
```

Comment:
And we had to comment the service output as the option `append` is only available on version  >240
```suggestion
# Runs on ubuntu 18, so systemd < 240, so 'file' output is broken:
# it rewrites the file from the beginning, appending is not available
#StandardOutput=file:/data/<%= @app_name %>/current/log/sidekiq_<%= @count %>.log
#StandardError=file:/data/<%= @app_name %>/current/log/sidekiq_<%= @count %>.stderr.log
```

Reply:
Good catch, file does output but as stated it would rewrite every time sidekiq is restarted so basically every deploy. Utilising append in later version of Ubuntu sounds like a plan though.